### PR TITLE
Allows duplicates of shuttle templates

### PR DESCRIPTION
**Pull request by ninjanomnom**
Pods aren't the only kind of shuttle we may want multiple of on a map.

### CODE REVIEW

File: code/modules/shuttle/computer.dm [hunk 1]
```diff
@@ -69,3 +69,6 @@
 	obj_flags |= EMAGGED
 	to_chat(user, "<span class='notice'>You fried the consoles ID checking system.</span>")
 
+/obj/machinery/computer/shuttle/proc/connect_to_shuttle(obj/docking_port/mobile/port, obj/docking_port/stationary/dock, idnum, override=FALSE)
+	if(port && (shuttleId = initial(shuttleId) || override))
```
Comment (SpaceManiac) [inline code/modules/shuttle/computer.dm]: `==` not `=`

File: code/modules/shuttle/shuttle.dm [hunk 2]
```diff
@@ -196,6 +196,30 @@
...
+	if(port.id == initial(port.id))
+		port.id = "[initial(port.id)][idnum]"
+	if(port.name == initial(port.name))
+		port.name = "[initial(port.name)] [idnum]"
+	for(var/i in port.shuttle_areas)
+		var/area/place = i
+		for(var/obj/machinery/computer/shuttle/comp in place)
+			comp.connect_to_shuttle(port, src, idnum)
+		for(var/obj/machinery/computer/camera_advanced/shuttle_docker/comp in place)
+			comp.connect_to_shuttle(port, src, idnum)
```
Comment (SpaceManiac) [inline code/modules/shuttle/shuttle.dm]: Is this where this code belongs? What about admin spawned shuttles?

File: code/modules/shuttle/shuttle.dm [hunk 3]
```diff
@@ -196,6 +196,30 @@
...
+	var/list/static/shuttle_id = list()
+	var/idnum
+	if(!shuttle_id[roundstart_template])
+		shuttle_id[roundstart_template] = idnum = 1
+	else
+		idnum = shuttle_id[roundstart_template]++
+	if(port.id == initial(port.id))
+		port.id = "[initial(port.id)][idnum]"
+	if(port.name == initial(port.name))
+		port.name = "[initial(port.name)] [idnum]"
```
Comment (SpaceManiac) [inline code/modules/shuttle/shuttle.dm]: Will this end up naming every single shuttle "NT Recovery Ship 1" etc.?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/shuttle.dm (hunk 3 above)
Comment (ninjanomnom) [reply code/modules/shuttle/shuttle.dm]: every shuttle? no

**Comment by ninjanomnom** [comment]
Got home and was able to check it works in game

**Pull request merged**